vulcan/hebe/data/_attendance.py

```diff
@@ -0,0 +1,150 @@
+# -*- coding: utf-8 -*-
+from typing import AsyncIterator, Union, List
+import datetime
+
+from related import (
+    immutable,
+    IntegerField,
+    StringField,
+    FloatField,
+    ChildField,
+    BooleanField,
+)
+
+from .._api_helper import FilterType
+from .._endpoints import DATA_ATTENDANCE
+from ..model import (
+    Serializable,
+    DateTime,
+    Teacher,
+    Subject,
+    Period,
+    TimeSlot,
+    TeamClass,
+    TeamVirtual,
+)
+
+
+@immutable
+class PresenceType(Serializable):
+    """Presence type
+
+    :var int ~.id: attendance ID
+    :var str ~.name: attendance name
+    :var str ~.symbol: attendance symbol
+    :var int ~.category_id: attendance category ID
+    :var str ~.category_name: attendance category name
+    :var int ~.position: attendance position
+    :var bool ~.presence: presence on lesson
+    :var bool ~.absence: absence on lesson
+    :var bool ~.exemption: exemption from lesson
+    :var bool ~.late: is late for lesson
+    :var bool ~.justified: justified absence
+    :var bool ~.deleted: whether the entry is deleted
+    """
+
+    id = IntegerField(key="Id")
+    name: str = StringField(key="Name")
+    symbol: str = StringField(key="Symbol")
+    category_id: int = IntegerField(key="CategoryId")
+    category_name: str = StringField(key="CategoryName")
+    position: int = IntegerField(key="Position")
+    presence: bool = BooleanField(key="Presence")
+    absence: bool = BooleanField(key="Absence")
+    exemption: bool = BooleanField(key="LegalAbsence")
+    late: bool = BooleanField(key="Late")
+    justified: bool = BooleanField(key="AbsenceJustified")
+    deleted: bool = BooleanField(key="Removed")
+
+
+@immutable
+class Attendance(Serializable):
+    """A grade.
```

Comment:
This is certainly not a grade :) 
Also, please remove the unneeded line breaks at the end of the docstring.